crate/cli/src/actions/shared/export_key.rs

```diff
@@ -24,6 +25,12 @@ pub enum ExportKeyFormat {
     Raw,
 }
 
+#[derive(clap::ValueEnum, Debug, Clone, PartialEq, Eq)]
+pub(crate) enum ExportBlockCipherMode {
```

Comment:
I wonder if we could use instead directly the KMIP BlockCipherMode enum (and add the nist-key-wrap as extension to it). Avoiding us to declare and maintain a new enum.

Reply:
I tried first - but wasn't able to access it from the CLI's folder; maybe we can watch this together next week